src/style.css

```diff
@@ -1,3 +1,62 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  list-style: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style elements using `.class`. In this case, `list-style: none;` should be applied to a specific class, such as `.nav__list`.